### DISCUSSION TITLE
What this issue is about?

### DISCUSSION THREAD

**Discussion by fernandofas** [prisma/prisma#13670] [Q&A]
I got this issue when running npx prisma generate

```
Error: Unable to require(/path to my directory/server/node_modules/prisma/libquery_engine-debian-openssl-3.0.x.so.node)
/snap/core20/current/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /lib/x86_64-linux-gnu/libcrypto.so.3)

I have:
"dependencies": {
"@prisma/client": "^3.14.0",
"apollo-server": "^2.19.0",
"bcryptjs": "^2.4.3",
"jsonwebtoken": "8.5.1"
},
"devDependencies": {
"nodemon": "^2.0.6",
"prisma": "^3.14.0"
}
```

OpenSSL3.0.2, node 16, ubuntu 22.04, all super updated, still not running.

Any idea how to sort this out?

**Comment by austincrim** [comment]
Hey @fernandofas!

My colleague recommended you try installing NodeJS with `apt` instead of `snap`.

Could you give that a try and let me know how it works?

**Reply by fernandofas** [reply]
@austincrim , please advice of how to do it please. Thank you

**Reply by pantharshit00** [reply]
@fernandofas Do thing following:
```
sudo snap remove node
sudo apt update
curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
sudo apt install -y nodejs
```

This will remove Node install via snap and install it from official Node.js apt repositories.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
